screwdriver/is_whitelisted.bash

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+PUBLISH_WHITELIST="master sd-master"
+
+# Only publish whitelisted branches
+echo "Branch: ${GIT_BRANCH} Whitelist: ${PUBLISH_WHITELIST}"
+WHITELISTED="false"
+for a in ${PUBLISH_WHITELIST}; do
+    if [[ ${GIT_BRANCH} = "origin/${a}" ]];
+    then
+      WHITELISTED="true";
```

Comment:
given that you are in a for loop, you might was well `break` here when the condition is true.